xml/en/docs/contributing_changes.xml

```diff
@@ -142,8 +146,27 @@ submitted as a pull request.
 <section id="license" name="License">
 
 <para>
-Submitting changes implies granting project a permission to use it under
-an appropriate <link url="../../LICENSE">license</link>.
+Submitting changes implies granting project a permission
+to use it under an appropriate
+<link url="https://github.com/nginx/nginx/blob/main/LICENSE">license</link>.
+</para>
+
+</section>
+
+
+<section id="see_also"
+         name="See also">
+
+<para>
+<list type="bullet">
+
+<listitem>
+<para>
+<link url="https://github.com/nginx/nginx/blob/master/CONTRIBUTING.md">Contributing Guidelines</link>
```

Comment:
This content duplicates this page and nginx.org documentation.
Although It can be useful standalone for those participants directly cloning repo with `git clone` without prior reading of nginx.org, it makes no sense to place the link here.